srcpkgs/radare2/template

```diff
@@ -1,19 +1,32 @@
 # Template file for 'radare2'
-# cutter needs to be bumped when radare2 is updated
+# iaito needs to be bumped when radare2 is updated
```

Comment:
We don't have iaito in our repositories. Did you know about #33608?

Reply:
And what should happen to cutter?

Reply:
> We don't have iaito in our repositories. Did you know about #33608?

Yes, iaito currently can't build against r2 so I didn't include it. I left the comment in for the future, can be removed if you'd rather.

> And what should happen to cutter?

Cutter has already been updated and no longer depends on r2 (#35901).